README.md

```diff
@@ -51,29 +42,39 @@ npm install
 ember serve
 ```
 
-Visit [http://localhost:4200](http://localhost:4200)
+Afterwards, visit [http://localhost:4200](http://localhost:4200) on your browser.
+
+Note: On Mac, if you get the error `Error: EMFILE: too many open files, watch`, try installing Watchman. Install [Homebrew](https://brew.sh/) if you don't have it. Then, in your terminal, run `brew install watchman`.
 
-Note: If you get an error on macOS mentioning `"Error: EMFILE: too many open files, watch"`, try installing Watchman. Install [Homebrew](https://brew.sh/) if you don't have it. Then in you terminal `brew install watchman`.
 
 ## Running tests
 
-Use `npm` to run tests instead of `ember`, since we have additional
-tests like spellchecking that are not part of The Ember app's tests.
+Use `npm test` to run tests locally. In addition to the Ember app, we check for broken links.
 
-```
-npm install
+```bash
 npm test
 ```
 
 ### Linting and spellchecking
 
-The guides are spellchecked and linted for Markdown consistency. You can test your contributions by running `npm run lint:md`. Linting and spellchecking must pass or they will fail in Travis-CI.  See [CONTRIBUTING.md](CONTRIBUTING.md) for more information on linting and spellchecking.
+The guides are spellchecked and linted for Markdown consistency. You can check your edits by running,
 
+```bash
+npm run lint:md
+```
 
 ### Internal and external links
 
-Testing of internal and external links can be performed using three commands:
+You can find broken internal (relative) and external links by running 3 commands:
 
-1. `npm run test:node`.  Checks all relative links for all versions of the guides and runs all ither test scripts in the `node-tests` directory, except for those located in the `node-tests/local` sub-directory;
-1. `npm run test:node-local`. Checks all external links in the `guides/release` folder; and
-1. `npm run test:node-local-exclude-api-urls`.  Checks all external links except for links to the [API docs](https://api.emberjs.com).
+```bash
+# Check all internal links across all versions of the Guides
```

Comment:
We seem to have lost some info from what was originally written. Is the additional context no longer relevant?

Reply:
Yeah, I didn't feel confident with updating this section.

I think my updated comments for `npm run test:node-local` and `npm run test:node-local-exclude-api-urls` are correct.

I'm not sure if I understood the current comment for `npm run test:node` correctly. The part ``runs all ither test scripts in the `node-tests` directory`` didn't make sense to me. Do you, by chance, understand what this script would do?

Reply:
I believe `ither` is a typo and should be `other`. I think it runs all tests in the `node-tests` directory.

Reply:
Haha, that makes more sense. I kept reading it as `either` and couldn't understand what the command does. I'll update the comment for the command later at lunch.